src/CSET/recipes/generic_domain_mean_vertical_profile_series.yaml

```diff
@@ -0,0 +1,41 @@
+category: Quick Look
+title: Domain mean horizontal $VARNAME vertical profile as series
+description: Plots a time series of the domain mean surface $VARNAME.
```

Comment:
The description needs to be update for vertical profiles.

Reply:
Changed to:
title: Domain mean $VARNAME vertical profile as series
description: Plots a time series of vertical profiles for the domain mean $VARNAME using a log pressure coordinate.